app/views/users/index.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<h1>Users</h1>
+<div class="user-container">
+<% @users.each do |user| %>
+  <div class="user-card">
+    <div class="user-image">
+          <img src="<%= user.photo_link %>" alt="<%= user.name %> photo" class="img-fluid rounded-circle">
+      </div>
+    <div class="user-info">
+      <h3><%= link_to user.name, user_path(user) %></h3>
+      <p class="comment">Number of posts: <%= user.posts.count %></p>
```

Comment:
- [ ] After I used the console to add data, I could not see it render the users even though all the data successfully added. Kindly quickly check it out 
![image](https://github.com/hydaline-code/myblogapp/assets/86200356/43b68069-e6bc-4289-9307-158ff30398de)